src/hooks/useAccommodationsAndOffers.tsx/api.ts

```diff
@@ -39,7 +37,9 @@ export async function fetchAccommodationsAndOffers({
    */
   const { data: coordinatesData } = await axios
     //TODO: include endpoint in .env or config
-    .get(`https://nominatim.openstreetmap.org/search?format=json&q=${location}`)
+    .get(`https://nominatim.openstreetmap.org/search?format=json&q=${location}`, {
+      withCredentials: true
```

Comment:
should it also be `true` for the open street map?

Reply:
no, you are right. I will remove it